### PR TITLE
IS- Student Payment Page

### PR DESCRIPTION
**Pull request by joselineem**
### Changes:
- Adding page for student payments history

### Evidence:
image
image

image
image

### CODE REVIEW

File: src/pages/Student/PaymentContent/PaymentContent.js [hunk 1]
```diff
@@ -1,17 +1,174 @@
...
+      <b className='space'>Estado: </b>{' '}
+      <span className='space dot'>
+        {statusObj.name}
+        {statusObj?.icon}
+      </span>{' '}
+    </div>
+  );
+  const price = (
+    <>
+      <b>Costo: </b> {row?.unitPrice || 'No disponible'}
```
Comment (AntonioChavezDev) [inline src/pages/Student/PaymentContent/PaymentContent.js:62]: Can we add a currency format for the costo field please?

Something like $500 MXN would be ideal at this point, nothing too complicated

**Review by AntonioChavezDev — CHANGES_REQUESTED** [review]
Great job!!
You're awesome!

I just left you a minor change regarding currency format for cost field, aside from that everything looks great!!

File: src/pages/Student/PaymentContent/PaymentContent.js (hunk 1 above)
Comment (joselineem) [reply src/pages/Student/PaymentContent/PaymentContent.js]: Done 👍🏻

**Review by AntonioChavezDev — APPROVED** [review]
Well done!!
Really appreciate it!

**Pull request merged**